backend/server/controllers/users.js

```diff
@@ -245,5 +245,44 @@ module.exports = {
         res.status(400).send('Error removing assistant.')
       }
     }
+  },
+
+  async removeStudent(req, res) {
+    if (!helper.controllerBeforeAuthCheckAction(req, res)) {
+      return
+    }
+    if (req.authenticated.success) {
+      try {
+        const userId = req.decoded.id
+        const siToRemove = await StudentInstance.findOne({
+          where: {
+            id: req.body.id,
+            userId
+          }
+        })
```

Comment:
Maybe teachers should also be able to remove mistaken registrations?

Reply:
Sounds reasonable, I'll change it later